interview_db/models.py

```diff
@@ -53,15 +51,15 @@ class Meta:
 
 
 class Student(models.Model):
-
     first_name = models.CharField(max_length=255)
     last_name = models.CharField(max_length=255, blank=True, null=True)
     uw_netid = models.CharField(max_length=10, blank=True, null=True)
     email = models.EmailField(max_length=255,
                               blank=True,
                               null=True)
     artifacts_url = models.URLField(
```

Comment:
Is there a Jira project for this? There's a task down the road that should be done to change how this is handled.